src/full/Agda/Interaction/Highlighting/HTML.hs

```diff
@@ -148,12 +148,13 @@ generatePage renderPage dir mod = do
   f   <- fromMaybe __IMPOSSIBLE__ . Map.lookup mod <$> useTC TCM.stModuleToSource
   css <- fromMaybe defaultCSSFile . optCSSFile <$> TCM.commandLineOptions
   pc  <- optHTMLOnlyCode <$> TCM.commandLineOptions
+  ext <- optHTMLOutputExt <$> TCM.commandLineOptions
```

Comment:
```suggestion
  ext <- optHTMLOutputExt <$> TCM.commandLineOptions
  let target = dir </> modToFile mod ext
```
> src/full/Agda/Interaction/Highlighting/HTML.hs:157:40: error:
>     • Variable not in scope: ext :: String
>     • Perhaps you meant one of these:
>         ‘text’ (imported from Text.Blaze.Html5)
>         ‘text’ (imported from Agda.Utils.Pretty)
>         ‘exp’ (imported from Prelude)

In case you didn't figure it out already, the where block only has the function arguments (variables left of the equals sign), global definitions and other definitions in the `where` block in scope. The variable `ext` is bound in the body of the function (right side of the equals sign), so it's not in scope in the `where` block.

Instead, you can use a `let` declaration just after the line where `ext` is bound.